app/models/manager_refresh/inventory/persister.rb

```diff
@@ -172,32 +172,28 @@ def to_raw_data
     }
   end
 
-  class << self
-    protected
+  def self.from_raw_data(persister_data)
```

Comment:
probably leave it under class << self, or move it up, to avoid the rubocop failure